src/containers/Tenant/Diagnostics/Configs/Configs.tsx

```diff
@@ -0,0 +1,159 @@
+
+import {PersonPencil} from '@gravity-ui/icons';
+import type {Column} from '@gravity-ui/react-data-table';
+import {Icon, Popover, Switch} from '@gravity-ui/uikit';
+import {StringParam, useQueryParam} from 'use-query-params';
+
+import {ResizeableDataTable} from '../../../../components/ResizeableDataTable/ResizeableDataTable';
+import {Search} from '../../../../components/Search';
+import {TableWithControlsLayout} from '../../../../components/TableWithControlsLayout/TableWithControlsLayout';
+import {setFeatureFlagsFilter, tenantApi} from '../../../../store/reducers/tenant/tenant';
+import type {FeatureFlagConfig} from '../../../../types/api/featureFlags';
+import {cn} from '../../../../utils/cn';
+import {DEFAULT_TABLE_SETTINGS} from '../../../../utils/constants';
+import {useTypedDispatch, useTypedSelector} from '../../../../utils/hooks';
+
+import i18n from './i18n';
+
+import './Configs.scss';
+
+const FEATURE_FLAGS_COLUMNS_WIDTH_LS_KEY = 'featureFlagsColumnsWidth';
+
+const b = cn('ydb-diagnostics-configs');
+
+const columns: Column<FeatureFlagConfig>[] = [
+    {
+        name: 'Touched',
+        header: '',
+        render: ({row}) =>
+            row.Current ? (
+                <Popover
+                    content={i18n('flag-touched')}
+                    className={b('icon-touched')}
+                    placement="left"
+                >
+                    <Icon data={PersonPencil} />
+                </Popover>
+            ) : null,
+        width: 36,
+        sortable: false,
+        resizeable: false,
+    },
+    {
+        name: 'Name',
+        get header() {
+            return i18n('td-feature-flag');
+        },
+        render: ({row}) => (row.Current ? <b>{row.Name}</b> : row.Name),
+        width: 400,
+        sortable: true,
+        sortAccessor: ({Current, Name}) => {
+            return Number(!Current) + Name.toLowerCase();
+        },
+    },
+    {
+        name: 'Default',
+        get header() {
+            return i18n('td-default');
+        },
+        render: ({row}) => {
+            switch (row.Default) {
+                case true:
+                    return i18n('enabled');
+                case false:
+                    return i18n('disabled');
+                default:
+                    return '-';
+            }
+        },
+        width: 100,
+        sortable: false,
+        resizeable: false,
+    },
+    {
+        name: 'Current',
+        get header() {
+            return i18n('td-current');
+        },
+        render: ({row}) => <Switch disabled checked={(row.Current ?? row.Default) || false} />,
+        width: 100,
+        sortable: false,
+        resizeable: false,
+    },
+];
+
+interface FeatureFlagsListsProps {
+    isError?: boolean;
+    featureFlags: FeatureFlagConfig[];
+    featureFlagsFilter?: string;
+}
+
+const FeatureFlagsList = ({isError, featureFlags, featureFlagsFilter}: FeatureFlagsListsProps) => {
+    if (isError) {
+        return i18n('error-msg');
```

Comment:
Why not a real error from query?
```suggestion
        return <ResponseError error={error} />;
```